tests/beacon/conftest.py

```diff
@@ -349,3 +363,25 @@ def max_attestation_count():
 @pytest.fixture
 def initial_fork_version():
     return SERENITY_CONFIG.INITIAL_FORK_VERSION
+
+
+#
+# genesis
+#
+@pytest.fixture
+@to_tuple
+def genesis_validators(init_validator_keys,
+                       init_randao,
+                       deposit_size):
+    return [
+        ValidatorRecord(
+            pubkey=pub,
+            withdrawal_credentials=ZERO_HASH32,
+            randao_commitment=init_randao,
+            randao_skips=0,
+            balance=deposit_size * denoms.gwei,
```

Comment:
question:  why is the deposit balance in gwei?  It seems like using `wei` would be the least error prone approach.

Reply:
Because we currently use `uint64` (~1.8e+19) for almost all number fields. The `MAX_DEPOSIT` is `32`, plus the validator will get rewards, so we utilize gwei (1e+18) as the unit of balance.

Reply:
So this lets the number fit into a `uint64` where-as if it was denominated in wei it would not?

Reply:
Correct. Per homogenizing types to int64 (ethereum/eth2.0-specs#36, ethereum/eth2.0-specs#48).

Reply:
Also, I assume your previous comment should have been `MIN_DEPOSIT`?